src/status_im/ui/screens/chat/stickers/views.cljs

```diff
@@ -64,12 +64,16 @@
 
 (defn update-scroll-position [ref installed-packs selected-pack window-width]
   (when ref
-    (let [x (if (= selected-pack :recent)
-              0
-              (* (inc (some #(when (= selected-pack (:id (second %))) (first %))
-                            (map-indexed vector installed-packs)))
-                 window-width))]
-      (.scrollTo ref (clj->js {:x x :animated true})))))
+    ;; bug on Android https://github.com/facebook/react-native/issues/24531
+    (js/setTimeout
```

Comment:
have you tried not update scroll position in component-will-update since it is deprecated anyway? instead do it in `component-did-update` (L85)